photutils/psf/epsf.py

```diff
@@ -293,41 +283,36 @@ class EPSFBuilder:
         `~astropy.modeling.fitting.LevMarLSQFitter`.  See the
         `EPSFFitter` documentation its options.
 
-    center_accuracy : float, optional
-        The desired accuracy for the centers of stars.  The building
-        iterations will stop if the centers of all the stars change by
-        less than ``center_accuracy`` pixels between iterations.  All
-        stars must meet this condition for the loop to exit.  The
-        default is 1.0e-3.
-
     maxiters : int, optional
-        The maximum number of iterations to perform.  If the
-        ``center_accuracy`` is met, then the iterations will stop prior
-        to ``maxiters``.  The default is 10.
+        The maximum number of iterations to perform. The default is 10.
 
     progress_bar : bool, option
         Whether to print the progress bar during the build iterations.
         The default is `True`.
     """
 
-    def __init__(self, oversampling=4., shape=None, smoothing_kernel='quartic',
-                 recentering_func=centroid_com, recentering_boxsize=(5, 5),
-                 recentering_maxiters=20, fitter=EPSFFitter(), center_accuracy=1.0e-3,
-                 maxiters=10, progress_bar=True):
+    def __init__(self, oversampling=4., shape=None,
+                 smoothing_kernel='quartic', norm_radius=5.5, shift_val=0.5,
```

Comment:
This doesn't look OK, order of arguments shouldn't change, new args should go to the end of the list.

Reply:
Sorry, thanks for letting me know! This must have slipped past in review. I'll add a small PR fix to move the new arguments to the back of the queue later; is it just `norm_radius` and `shift_val` or are there any other issues with the new ordering I should fix while I'm there?